jwst/tests_nightly/general/miri/test_linearity3.py

```diff
@@ -37,7 +37,7 @@ def test_linearity_miri3(_bigdata):
     newhref = pf.HDUList([href['primary'],href['sci'],href['err'],href['pixeldq'],href['groupdq']])
     result = pf.diff.FITSDiff(newh,
                               newhref,
-                              ignore_keywords = ['DATE','CAL_VER','CAL_VCS','CRDS_VER','CRDS_CTX'],
+                              ignore_keywords = ['DATE','CAL_VER','CAL_VCS','CRDS_VER','CRDS_CTX', 'R_LINEAR'],
```

Comment:
Why would you want to ignore the name of the linearity reference file that was used? Comparison against that keyword value will tell you when a new/different reference file has been used. That way you can confirm changes to CRDS and also then know why the image pixel values have changed.

Reply:
The problem is that the way the test is written is that this is machine-dependent.  It fails if we allow the comparison:
```
E            Headers contain differences:
E              Keyword R_LINEAR has different values:
E                 a> /Users/jdavies/jwst_test_data/miri/test_linearity/lin_nan_flag_miri.fits
E                  ?  -------  ^^^^
E                 b> /data4/jwst_test_data/miri/test_linearity/lin_nan_flag_miri.fits
E                  ?    ^^^
E              Keyword R_LINEAR has different comments:
E                 a> Linearity reference file name
E                 b> L
```
This is a general problem with absolute paths in our testing environment.  We need to figure out a way to get away from this.  Any ideas?

Reply:
For reasons of privacy and security, I think only the file name should be included in the metadata for the file.  The directory path should be omitted.  That would also help with the problem of strings that are too long for the archive.

Reply:
Oh, I see. I didn't realize/remember that some tests were using custom reference files, as opposed to the latest/greatest from CRDS. Of course you still don't really need to skip the comparison of the keyword value; just update the comparison file so that it has the keyword value with the new path.

@philhodge I see the point about privacy, in terms of paths that are personal directories, but there's also the flip side of a user doing custom work outside of the pipeline wanting to have complete, explicit history of what was used in their processing. In the Ops environment, which are the only files that will ever go into the archive catalog, the path is the standard "crds://", so there's no privacy issue and it's also a short string.

Reply:
I will back out this change in a subsequent PR @hbushouse.  And we can deal with this issue in the pipeline code.